modules/dev-tools/package.json

```diff
@@ -16,6 +16,7 @@
     "templates"
   ],
   "bin": {
+    "ocular-bootstrap-monorepo": "./scripts/bootstrap-monorepo.sh",
```

Comment:
I think we could just call it `ocular-bootstrap` and following your directory sniffing autodetection logic, just do nothing if not a monorepo....